tests/base.py

```diff
@@ -0,0 +1,93 @@
+import contextlib
+import unittest
+
+import wrapt
+
+from ddtrace import config
+
+from .utils.tracer import DummyTracer
+from .utils.span import TestSpanContainer, TestSpan
+
+
+class BaseTestCase(unittest.TestCase):
+    """
+    BaseTestCase extends ``unittest.TestCase`` to provide some useful helpers/assertions
+
+
+    Example::
+
+        from tests import BaseTestCase
+
+
+        class MyTestCase(BaseTestCase):
+            def test_case(self):
+                self.assert_is_wrapped(obj)
+    """
+    def assert_is_wrapped(self, obj):
```

Comment:
yeah, this is one I felt probably fit into the base test case, since we might need to check for wrapping just about anywhere given the nature of our code.

Happy to move to a `PatchMixin` or something for now and add back here if need be or something.

No strong preference.